### PR TITLE
feat: no need to stringify props for android video

### PR DESCRIPTION
**Pull request by dickie81**
## Description:
Video properties auto stringified for android.

## Known Issues:
None

### Checklist for this pull request

* [X] PR is scoped to one task
* [ ] Tests are included
* [X] Documentation included
* [X] One of:
  * [X] The PR is not making any UI change
  * [ ] The PR is making a UI change but not a product change
    * [ ] Screenshots included

Thank you!

### CODE REVIEW

File: src/player/APVideoPlayer.js [hunk 1]
```diff
@@ -43,8 +48,24 @@ const ReactVideoView = requireNativeComponent(
...
+  src: { type, object, player_configuration, startTime },
+  maxWidth,
+  ratio,
+  style
+}) => {
   const dims = getFrameDims(maxWidth, ratio);
+
+  const src = {
+    type,
+    object: Platform.select({ android: JSON.stringify(object), ios: object }),
```
Comment (Budaa) [inline src/player/APVideoPlayer.js:62]: We should implement type checking before stringifying to ensure backward compatibility and make sure we don't double stringify.

**Review by Budaa — CHANGES_REQUESTED** [review]

File: src/player/APVideoPlayer.js (hunk 1 above)
Comment (dickie81) [reply src/player/APVideoPlayer.js:62]: @Budaa, I have just added more specific proptypes validation to better indicate what is expected as input. I made the commit a breaking change so the version should be bumped on NPM. I don't think that supporting fuzzy proptypes makes sense, better if people remove their android work arounds when updating this module.

Comment (Budaa) [reply src/player/APVideoPlayer.js:62]: Fair enough for me 👍

**Review by Budaa — DISMISSED** [review]

**Review by Budaa — APPROVED** [review]
LGTM

**Pull request merged**